src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './CommentInfo.scss';
+
+// type Props = {
+//   comments: Comment;
+// };
+
+// export const CommentInfo: React.FC<Props> = (props) => {
+//   const { comments } = props;
+
+//   return (
+//     <div className="comment">
+//       <p>
+//         User name:
+//         {' '}
+//         {comments.name}
+//       </p>
+//       <p>
+//         Comment:
+//         {' '}
+//         {comments.body}
+//       </p>
+//       <p>
+//         Email address:
+//         {' '}
+//         {comments.email}
+//       </p>
+//     </div>
+//   );
+// };
+
+export const CommentInfo: React.FC<Comment> = ({ name, body, email }) => {
```

Comment:
It'd better to use variant that you have in comments. Destructuring is better to be done in fuction body
```suggestion
export const CommentInfo: React.FC<Props> = (props) => {
```